guardrails/cli/configure.py

```diff
@@ -64,8 +59,27 @@ def configure(
         help="Clear the existing token from the configuration file.",
     ),
 ):
-    if clear_token is True:
+    existing_token = _get_default_token()
+    last4 = existing_token[-4:] if existing_token else ""
+
+    if not clear_token:
+        console.print("\nEnter API Key below", style="bold", end=" ")
+        last4 and console.print(
+            "[dim]leave empty if you want to keep existing token[/dim]",
+            style="italic",
+            end=" ",
+        )
+        last4 and console.print(f"[{last4}]", style="italic")
+
+        console.print(
+            ":backhand_index_pointing_right: You can find your API Key at https://hub.guardrailsai.com/keys"
```

Comment:
I'm getting a 404 when I copy/paste this.

Reply:
Perhaps "https://hub.guardrailsai.com/tokens" ?

Reply:
ah yes, there's actually a URL change here with the new API keys page. Currently it's `https://hub.guardrailsai.com/tokens` but will be `https://hub.guardrailsai.com/keys`. 

I think the new api keys page will get rolled out in the coming days (prior to this), but not opposed also to making sure we have some redirect just in case

Reply:
Good idea.